JavaScript/JavaScriptSDK/Context/User.ts

```diff
@@ -29,12 +33,60 @@ module Microsoft.ApplicationInsights.Context {
          * The store region.
          */
         public storeRegion: string;
-        
+
+        /**
+         * Authorized user id
+         */
+        private authId: string;
```

Comment:
@Assaf-Neufeld  +1 to Oleg's comments - please don't add getAuthId(), this introduces API inconsistency. Instead let's wait until it's obvious that this getter needs to be added.